trip_app/accounts/tests/user_factory.py

```diff
@@ -8,7 +8,7 @@
 
 class UserFactory(factory.django.DjangoModelFactory):
     """User factory."""
-    username = factory.LazyAttribute(lambda _: faker.user_name())
+    username = factory.Sequence(lambda n: 'user%d' % n)
```

Comment:
Sequence should deliver uniqueness of this field and help avoid potential issues connected with the violence of this rule.

Reply:
_Violation_ you mean? :D
And yes, it makes sense.

Reply:
Ups, that is exactly what I meant. Peace :grin: